docs/docs/metrics.md

```diff
@@ -117,24 +94,23 @@ Please note that prior to <span class="label label-default">v0.15</span> these w
 number of HTTP requests received by Marathon is available under `.count`.
 There are more metrics around HTTP requests under the
 `org.eclipse.jetty.servlet.ServletContextHandler` prefix.
-For more infos, look at
+For more information, look at
 [the code](https://github.com/dropwizard/metrics/blob/796663609f310888240cc8afb58f75396f8391d2/metrics-jetty9/src/main/java/io/dropwizard/metrics/jetty9/InstrumentedHandler.java#L41-L42).
```

Comment:
It is the best documentation we have (there is no other public documentation). If you think this is misleading - please remove the link.